stingray/modeling/posterior.py

```diff
@@ -333,6 +333,33 @@ def evaluate(self, pars, neg=False):
         else:
             return loglike
 
+class PSDLogLikelihoodBound(PSDLogLikelihood):
+    def __init__(self, *args, **kwargs):
```

Comment:
In what way does `PSDLogLikelihoodBound` differ from `PSDLogLikelihood`? I can't really see that.